crates/sats/src/lib.rs

```diff
@@ -177,3 +188,30 @@ impl<'a, T: ?Sized> WithTypespace<'a, T> {
         }
     }
 }
+
+pub struct IterWithTypespace<'a, I> {
+    typespace: &'a Typespace,
+    iter: I,
+}
+
+impl<'a, I, T: 'a> Iterator for IterWithTypespace<'a, I>
+where
+    I: Iterator<Item = &'a T>,
+{
+    type Item = WithTypespace<'a, T>;
```

Comment:
Are these changes related to dependency update or can they be split out too as separate refactoring?

Reply:
I did this as I was changing the table display code to work with the new version of tabled; technically it could be split out but if it's alright I'd rather just not mess with it (and have to separate the tabled changes from the iter changes)? But if you feel strongly about it I could split it off.

Reply:
Not very strongly since it's related to dependency update. If it was completely unrelated, I'd rather keep it separate - more focused PRs are easier to review - but sounds like it's not.